src/app/organizations/organizations/organizations.component.html

```diff
@@ -44,9 +44,9 @@
         </mat-form-field>
         <mat-form-field appearance="outline" class="search-form-field">
           <mat-label>Search Organizations</mat-label>
-          <input #search matInput formControlName="name" />
+          <input #search matInput formControlName="name" [(ngModel)]="search.value" />
           <button mat-button *ngIf="search.value" matSuffix mat-icon-button (click)="search.value = ''">
-            <mat-icon inline="true">close</mat-icon>
+            <mat-icon style="font-size: 16px">close</mat-icon>
```

Comment:
inline style https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist

Reply:
Actually changed the size back to the original to match the sizes of the search page and home page, but could take suggestions and opinions if the smaller close icon looks better on the organization page.